src/transpose/index.js

```diff
@@ -4,6 +4,25 @@
  * @param {number[]} array The array to transpose
  * @returns {number[]} The transposed array
  */
-function transpose(array) {}
+
+
+function transpose(array) {
+    return array.reduce((prev, next) => next.map((item, i) => (prev[i] || []).concat(next[i])), []);
+}
+
 
 module.exports = transpose;
+
+
+
+
+
+
+
+
+/*
+return array.reduce((prev, next)=> 
+    next.map((item, i)=> 
+    (prev[i] || []).concat(next[i])), [])
+
+*/
```

Comment:
Guess you forgot to remove this.

Reply:
Thanks for point that out.
I totally forgot. I needed to do a rearrangement of the code block for better result